src/FieldArray.js

```diff
@@ -75,7 +75,8 @@ class FieldArray extends React.PureComponent<Props, State> {
       listener,
       subscription ? { ...subscription, length: true } : all,
       {
-        getValidator: () => this.validate
+        getValidator: () => this.validate,
+        isEqual: () => true
```

Comment:
This is not  allowing a custom isEqual function, it is just returning true for any array level dirty check. I would say it should be more on the lines of this https://github.com/anant-singh/react-final-form-arrays/blob/dirty-field-check/src/FieldArray.js#L69

Reply:
what makes you think we should allow a custom isEqual function?
The purpose is to always return true because separate fields inside this array will be checked separately anyway. I described it [here as the first option](https://github.com/final-form/react-final-form/issues/151#issuecomment-421644035)

Reply:
It gives the option to the user to use per field check or pass a custom isEqual function at the array level. In my use case, I actually need to know if the array is dirty and if I put a isEqual on fields of an array, in the dirtyfields the field registered would be `array[0]`, whereas on a array level check, I get `array`.

Reply:
yeah, you're right @anant-singh. I'm going to refactor it today.